target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -582,6 +583,7 @@ ath79_setup_macs()
 		label_mac=$wan_mac
 		;;
 	ubnt,litebeam-ac-gen2|\
+	ubnt,nanobeam-ac-gen2|\
```

Comment:
Did you verify this?

Reply:
Yes. I tried the call `mtd_get_mac_binary art 0x5006` that gave me some useful stuff back.

I tried again `caldata_extract "art" 0x5000 0x844` and it was crashing my nanobeam ac gen2?

Reply:
art 0x5006 shouldn't just be "useful", but matching the MAC address on the label ...?

Reply:
I mean it is beginning with `E0:63:DA` and that is ubiquiti networks. ;) I can not easily access the antenna, but maybe I can flash airos again and check mac address, then.

Reply:
flashing airos won't help you in order to find out what's printed on the label. If you are not sure what's on the label, don't set up a label mac address, and just remove that part.

Reply:
*removed* ;)

Reply:
added again, since I could verify that is matches with the label mac on the box in wihch it was delivered